e2e/tests/hmctsAdministersCaseAfterSubmission_@slow_test.js

```diff
@@ -57,51 +60,78 @@ Scenario('HMCTS admin amends children, respondents, others, international elemen
+  I.seeInTab(['Additional applications 1', 'C2 application', 'Supporting documents 1', 'File'], 'mockFile.txt');
+  I.seeInTab(['Additional applications 1', 'C2 application', 'Additional orders requested'], 'Parental responsibility');
+  I.seeInTab(['Additional applications 1', 'C2 application', 'Who\'s seeking parental responsibility?'], 'Parental responsibility by the father');
+
+  I.seeInTab(['Additional applications 1', 'Other applications', 'File'], 'mockFile.txt');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Application type'], 'C1 - Parental responsibility');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Who\'s seeking parental responsibility?'], 'Parental responsibility by the father');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Date and time of upload']);
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Uploaded by'], 'HMCTS');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Supplements 1', 'Document name'], 'C20 - Secure accommodation');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Supplements 1', 'Which jurisdiction?'], 'England');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Supplements 1', 'Notes'], 'This is a note about supplement');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Supplements 1', 'Date and time uploaded']);
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Supplements 1', 'File'], 'mockFile.txt');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Supplements 1', 'Uploaded by'], 'HMCTS');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Supporting documents 1', 'Document name'], 'Supporting document');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Supporting documents 1', 'Notes'], 'This is a note about supporting doc');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Supporting documents 1', 'Date and time uploaded']);
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Supporting documents 1', 'Uploaded by'], 'HMCTS');
+  I.seeInTab(['Additional applications 1', 'Other applications', 'Supporting documents 1', 'File'], 'mockFile.txt');
+
+  I.seeInTab(['Additional applications 1', 'PBA Payment', 'Paid with PBA'], 'Yes');
+  I.seeInTab(['Additional applications 1', 'PBA Payment', 'Payment by account (PBA) number'], c2Payment.pbaNumber);
+  I.seeInTab(['Additional applications 1', 'PBA Payment', 'Client code'], c2Payment.clientCode);
+  I.seeInTab(['Additional applications 1', 'PBA Payment', 'Customer reference'], c2Payment.customerReference);
 });
 
-Scenario('HMCTS admin edits supporting evidence document on C2 application', async({I, caseViewPage, manageDocumentsEventPage}) => {
+xScenario('HMCTS admin edits supporting evidence document on C2 application', async({I, caseViewPage, manageDocumentsEventPage}) => {
```

Comment:
Manage documents populates c2 documents from `c2DocumentBundle`. Populating c2 data from the new case field is out of scope for [FPLA-2744](https://tools.hmcts.net/jira/browse/FPLA-2744). It will be implemented in [FPLA-2763](https://tools.hmcts.net/jira/browse/FPLA-2763)